astropy/io/ascii/tests/test_cds.py

```diff
@@ -0,0 +1,441 @@
+ 25- 31  E7.1   ---    s       [-9e+34/2.0] Description of s     
+ 33- 35  I3     ---    i       [-30/67] Description of i         
+ 37- 39  F3.1   ---    sameF   [5.0/5.0] Description of sameF    
+ 41- 42  I2     ---    sameI   [20] Description of sameI         
+ 44- 47  F4.1   h      RAh     Right Ascension (hour)            
+ 49- 51  F3.1   min    RAm     Right Ascension (minute)          
+ 53- 70  F18.15 s      RAs     Right Ascension (second)          
+     72  A1     ---    DE-     Sign of Declination               
+ 73- 76  F5.1   deg    DEd     Declination (degree)              
+ 78- 81  F4.1   arcmin DEm     Declination (arcmin)              
+ 83- 98  F16.13 arcsec DEs     Declination (arcsec)              
+
+--------------------------------------------------------------------------------
+
+See also:
+
+
+Acknowledgements:
+
+References:
+================================================================================
+     (prepared by 1st author ?  / astropy.io.ascii )
+--------------------------------------------------------------------------------
+HD81809  1e-07  22.25608   2e+00  67 5.0 20 22.0 2.0 15.450000000007265 -61.0 39.0 34.5999960000006
+HD103095 -3e+06 27.25000  -9e+34 -30 5.0 20 22.0 2.0 15.450000000007265 -61.0 39.0 34.5999960000006
+''',  # noqa: W291
+
+positive_de = '''\
+--------------------------------------------------------------------------------
+Byte-by-byte Description of file: table.dat
+--------------------------------------------------------------------------------
+ Bytes Format Units  Label     Explanations
+--------------------------------------------------------------------------------
+  1-  8  A8     ---    names   Description of names              
+ 10- 14  E5.1   ---    e       [-3160000.0/0.01] Description of e
+ 16- 23  F8.5   ---    d       [22.25/27.25] Description of d    
+ 25- 31  E7.1   ---    s       [-9e+34/2.0] Description of s     
+ 33- 35  I3     ---    i       [-30/67] Description of i         
+ 37- 39  F3.1   ---    sameF   [5.0/5.0] Description of sameF    
+ 41- 42  I2     ---    sameI   [20] Description of sameI         
+ 44- 47  F4.1   h      RAh     Right Ascension (hour)            
+ 49- 52  F4.1   min    RAm     Right Ascension (minute)          
+ 54- 70  F17.14 s      RAs     Right Ascension (second)          
+ 72- 75  F4.1   deg    DEd     Declination (degree)              
+ 77- 80  F4.1   arcmin DEm     Declination (arcmin)              
+ 82- 99  F18.15 arcsec DEs     Declination (arcsec)              
+
+--------------------------------------------------------------------------------
+
+See also:
+
+
+Acknowledgements:
+
+References:
+================================================================================
+     (prepared by 1st author ?  / astropy.io.ascii )
+--------------------------------------------------------------------------------
+HD81809  1e-07  22.25608   2e+00  67 5.0 20 12.0 48.0 15.22440720000489 17.0 46.0 26.496624000004374
```

Comment:
At least one of the tests should have coordinates that are different in line 1 and line 2. Those are different stars, they in fact DO have different coordinates! For testing, we just want to make sure we would catch a bug that accidentally write the same value (e.g. the first) into every row.